specs-go/v1/image_index.go

```diff
@@ -50,9 +50,9 @@ type ManifestDescriptor struct {
 	Platform Platform `json:"platform"`
 }
 
-// ManifestList references manifests for various platforms.
-// This structure provides `application/vnd.oci.image.manifest.list.v1+json` mediatype when marshalled to JSON.
-type ManifestList struct {
+// ImageIndex references manifests for various platforms.
+// This structure provides `application/vnd.oci.image.index.v1+json` mediatype when marshalled to JSON.
+type ImageIndex struct {
```

Comment:
comment of  Annotations  `// Annotations contains arbitrary metadata for the manifest list.` below should change to `manifest list` to `image index`